must-gather/README.md

```diff
@@ -41,4 +41,37 @@ You will get a dump of:
     ```
 
 In order to get data about other parts of the cluster (not specific to ocs) you should
-run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
\ No newline at end of file
+run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
+
+### How to Contribute
+
+#### Contribution Flow
+Developers must follow these steps to make a change:
+1. Fork the `openshift/ocs-operator` repository on GitHub.
+2. Create a branch from the master branch, or from a versioned branch (such
+   as `release-4.2`) if you are proposing a backport.
+3. Make changes.
+4. Ensure your commit messages include sign-off.
+5. Push your changes to a branch in your fork of the repository.
+6. Submit a pull request to the `openshift/ocs-operator` repository.
+7. Work with the community to make any necessary changes through the code
+   review process (effectively repeating steps 3-7 as needed).
+
+#### Commit and Pull Request Messages
+
+- The message must begin with must-gather and a colon followed by single line description.
+- The message should continue with a longer description of the change.
+  - The description must be preceded by a single blank line to separate it from
+    the summary.
+  - It must describe why the change is necessary or useful.
+  - It must describe how the change was implemented.
+  - It must reference any open Issues the change addresses.
+  - It must wrap at 80 characters.
+  - *For very small changes it may be acceptable to omit the longer description.
+    Please remember, that it is easy for any developer to believe their code is
+    "self-documenting" when it is not. Adding a description will help future
+    maintainers of the project.*
+- The message must end with a sign-off
```

Comment:
when you finish writing all the points, add an example showing how a proper commit message looks like.
Give an example of how to signoff a PR
` git commit --signoff` and then how to write the full commit message finally.